### PR TITLE
Add function to show picker options menu

### PR DESCRIPTION
**Pull request by carcos-wolox**
## Summary ##

Add function to image picker that shows a menu with the options provided.

## Screenshots ##
image

### CODE REVIEW

**Review by danielaRiesgo — CHANGES_REQUESTED** [review]
I won't do it like this exactly.
You see we already have the `func presentImagePickerController(from source: UIImagePickerControllerSourceType, for media: [MediaPickerMediaType], onPermissionNotGranted: @escaping () -> Void)` public function.
I would change that so that it can receive a Set of UIImagePickerControllerSourceType. And if the array has only one element, do as we do now, and if it has more than one, then show the action sheet with the options and according to which one is chosen, then present the correct controller.
So in summary we would have something like this:
```
public protocol MediaPickerServiceType {
    var mediaSignal: Signal<MediaPickerMedia, MediaPickerServiceError> { get }

    func presentImagePickerController(from source: UIImagePickerControllerSourceType,
                                      for media: [MediaPickerMediaType],
                                      onPermissionNotGranted: @escaping () -> Void)

    func presentImagePickerController(from sources: [UIImagePickerControllerSourceType],
                                      for media: [MediaPickerMediaType],
                                      onPermissionNotGranted: @escaping () -> Void)
    
    var cameraIsAvailable: Bool { get }
}
```

**Review by danielaRiesgo — COMMENTED** [review]
Those small things and then it's approved :D

**Comment by danielaRiesgo** [comment]
Also, let's make a refrence to issue #6 that is the one we are solving here.

File: Utils/Services/ImagePickerService.swift [hunk 1]
```diff
@@ -96,6 +111,37 @@ public final class MediaPickerService: NSObject, MediaPickerServiceType {
...
+        guard source.count > 1 else {
+            presentImagePickerController(from: source[0], for: media, onPermissionNotGranted)
+            return
+        }
+        
+        let actionSheet = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+        
+        for option in source {
+            let button = UIAlertAction(title: option.menuTitle, style: .default, handler: { [unowned self] (action) -> Void in
+                option.isPermitted().startWithResult { [unowned self] in
```
Comment (nahuelgl) [inline Utils/Services/ImagePickerService.swift]: this block inside of the closure is exactly the same as the other method, make a function so you dont repeat code

File: Utils/Services/ImagePickerService.swift [hunk 2]
```diff
@@ -66,6 +77,10 @@ public protocol MediaPickerServiceType {
 @objc
 public final class MediaPickerService: NSObject, MediaPickerServiceType {
     
+    struct Constants {
```
Comment (nahuelgl) [inline Utils/Services/ImagePickerService.swift]: use `static let` property directly, no need for a struct

**Review by nahuelgl — CHANGES_REQUESTED** [review]

**Review by nahuelgl — APPROVED** [review]

**Comment by nahuelgl** [comment]
@carcos-wolox why didnt you merge this?

**Pull request merged**